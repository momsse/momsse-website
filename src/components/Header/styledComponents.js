import styled from 'styled-components'
import Link from 'gatsby-link'

export const Container = styled.div`
  background-color: #a6b956;
  margin-bottom: 1.45rem;
`

export const Content = styled.div`
  margin: 0 auto;
  max-width: 960px;
  padding: 1.45rem 1.0875rem;
`

export const Title = styled.h1`
  margin: 0;
`

export const StyledLink = styled(Link)`
  color: white;
  text-decoration: none;
`
