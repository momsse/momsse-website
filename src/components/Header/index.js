import React from 'react'
import { Container, Content, StyledLink, Title } from './styledComponents'

const Header = ({ siteTitle }) => (
  <Container>
    <Content>
      <Title>
        <StyledLink to="/" >{siteTitle}</StyledLink>
      </Title>
    </Content>
  </Container>
)

export default Header
