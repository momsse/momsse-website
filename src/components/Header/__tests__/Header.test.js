import React from 'react'
import renderer from 'react-test-renderer'

import Header from '../index';

describe('components/<Header />', () => {
  it('should render successfully', () => {
    const tree = renderer.create(<Header siteTitle="Hello" />).toJSON()

    expect(tree).toMatchSnapshot()
  });
})
