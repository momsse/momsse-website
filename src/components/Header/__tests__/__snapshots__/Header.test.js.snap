// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/<Header /> should render successfully 1`] = `
.c0 {
  background-color: #a6b956;
  margin-bottom: 1.45rem;
}

.c1 {
  margin: 0 auto;
  max-width: 960px;
  padding: 1.45rem 1.0875rem;
}

.c2 {
  margin: 0;
}

.c3 {
  color: white;
  -webkit-text-decoration: none;
  text-decoration: none;
}

<div
  className="c0"
>
  <div
    className="c1"
  >
    <h1
      className="c2"
    >
      <Link
        className="c3"
        to="/"
      >
        Hello
      </Link>
    </h1>
  </div>
</div>
`;
